src/components/v2/Layout/Sidebar/styles.ts

```diff
@@ -0,0 +1,86 @@
+import { css } from '@emotion/react';
+import { useTheme } from '@mui/material';
+
+export const useStyles = ({ expanded }: { expanded: boolean }) => {
+  const theme = useTheme();
+  return {
+    drawer: css`
+      display: none;
+      ${theme.breakpoints.up('md')} {
+        display: block;
+      }
+    `,
+    toolbar: css`
+      display: flex;
+      justify-content: center;
+      padding-top: ${theme.spacing(4)};
+      padding-bottom: ${theme.spacing(3)};
+      ${theme.breakpoints.up('md')} {
+        min-height: 0;
+      }
+      ${theme.breakpoints.up('sm')} {
+        &.MuiToolbar-root {
+          padding-left: ${theme.spacing(1)};
+          padding-right: ${theme.spacing(1)};
+        }
+      }
+      .MuiButton-text {
+        padding: 0;
+        min-width: 0;
+      }
+    `,
+    list: css`
+      padding-top: ${theme.spacing(1.25)};
+      .activeMenuItem {
+        color: ${theme.palette.text.primary};
+        svg {
+          color: ${theme.palette.text.primary};
+        }
+      }
+    `,
+    listItem: css`
+      transition: color 0.3s;
+      color: ${theme.palette.text.secondary};
+      min-width: auto;
+      padding: 0;
+      a {
+        padding-left: ${theme.spacing(2)};
+        padding-right: ${theme.spacing(0.5)};
+        padding-top: ${theme.spacing(3)};
+        padding-bottom: ${theme.spacing(3)};
+        display: inline-flex;
+      }
+
+      .active-menu-item {
+        padding-left: ${theme.spacing(1.5)};
+        .left-border {
+          border: 2px solid ${theme.palette.interactive.primary};
+          border-radius: 0 ${theme.shape.borderRadius}px ${theme.shape.borderRadius}px 0;
```

Comment:
Since we have a lot of different border radiuses used in the new design, I wouldn't use the value set in the theme. I actually think we should get rid of it completely.

Reply:
I wouldn't get rid of it, I would add them all to the theme personally

Reply:
Ok, can you elaborate? That's a small thing so I'm cool with that, but it feels like the theme is there to contain values that can be updated dynamically, for example if we want to support a lightmode. Border radiuses won't ever be dynamically changed.

Reply:
Agree, that border radius should be described with theme variables.

Reply:
Why though?

Reply:
cause it can easily been changed in case of theme v3, for example